front-end/src/app/shared/models/contact.model.ts

```diff
@@ -0,0 +1,71 @@
+import { plainToClass } from 'class-transformer';
+import { BaseModel } from './base.model';
+import { LabelList } from '../utils/label.utils';
+
+export enum ContactTypes {
+  CANDIDATE = 'cand',
```

Comment:
Are these the values you were saying we'd need to use in the api?  What's the source of these?

Reply:
Those are not the correct values. I have the correct values in the branch I'm working on now. I will push at the end of the day and send you a link to see them.